src/JabbR-Core/project.json

```diff
@@ -12,7 +12,8 @@
     "Microsoft.Extensions.Logging.Console": "1.0.0",
     "Microsoft.AspNetCore.Mvc": "1.0.0",
     "Microsoft.AspNetCore.StaticFiles": "1.0.0",
-    "Microsoft.Extensions.Configuration.UserSecrets": "1.0.0"
+    "Microsoft.Extensions.Configuration.UserSecrets": "1.0.0",
```

Comment:
@davidfowl This enables the SecretsManager CLI tool for adding dev secrets as env variables. https://docs.asp.net/en/latest/security/app-secrets.html?highlight=user%20secrets